skll/config/__init__.py

```diff
@@ -292,7 +353,7 @@ def parse_config_file(config_path, log_level=logging.INFO):  # noqa: C901
     sampler : str
         The name of a sampler to perform non-linear transformations of the input.
 
-    fixed_sampler_parameters : dict
+    fixed_sampler_parameters : List[Dict[str, Any]]
         A dictionary containing parameters you want to have fixed for the sampler.
```

Comment:
is it a dictionary or a list of?

Reply:
It's a list of dictionaries. I will fix the docstring.